pandas/util/mixins.py

```diff
@@ -0,0 +1,28 @@
+"""
+Mixins which can't go to core.common or core.base due to circular imports
+"""
```

Comment:
The only way I could see to break this cycle is change the (private) API on AccessorProperty to get a callable, which returns the `StringMethod` class. But IMO that's really ugly...